src/config/global-config.c

```diff
@@ -654,6 +657,80 @@ int Read_Global(XML_NODE node, void *configp, void *mailp)
                 }
 
             }
+        }
+        /*Agent's disconnection time parameters */
```

Comment:
I think this comment should be replicated at least on the other new config. But it can be deleted.

Reply:
It was replicated.
They are just to improve clarity.